apps/framework-cli/src/project.rs

```diff
@@ -235,6 +235,15 @@ impl Project {
             api_file.write_all(api_file_content.as_bytes())?;
         }
 
+        let deno_consumption_helper = deno_dir.join(CONSUMPTION_HELPERS);
+
+        if !deno_consumption_helper.exists() {
```

Comment:
you may want to always override so that when we ship a new version you override the old one and update it